scripts/gen_universe.py

```diff
@@ -282,18 +277,9 @@ def read_package(path):
     :rtype: dict
     """
 
-    package_json_path = path / 'package.json'
-
-    with package_json_path.open(encoding='utf-8') as file_object:
-        package_json = json.load(file_object)
-
-        # Update the `lastUpdated` field in the `package.json` if it is not already present.
-        if (LooseVersion(package_json.get('packagingVersion', '0.0')) >= LooseVersion("3.0")) \
-                and ("lastUpdated" not in package_json):
-            last_updated = git_repo.log("-1", '--date=unix', '--format=%cd', '--', path)
```

Comment:
I still like the idea of getting the last updated time from the git repo where possible. Why not catch the exception and continue or check for the presence of a .git directory? we can still support the git repo approach for some cases

Reply:
+1 it would be good to still have the tooling to populate this value for SDK FWs to benefit future releases

Reply:
I thought the same too, but, the fact that this does NOT work all the time means there would be some ambiguity and we need to have a backup anyway. I feel not touching the `package.json` at all would be more ideal and feels like WYSIWYG. For certified packages, we would rely on dcos-commons tooling to inject the field so IMO that automation is good enough for now( i believe Marcin had a ticket for that)

Reply:
Also, this tooling is not sufficient for packages published at https://downloads.mesosphere.com/universe/packages/packages.html - which would be weird cuz the same version would have a lastUpdated in public universe but NOT in airgapped environment and I thought reverting the change would be better to be consistent.

Reply:
I don't really know how this local-universe build thing is supposed to work (is it the customer who runs it, or us, or both?), but either way we're missing some cases - unless updating the SDK tooling will be enough to make such packages as `dcos-ui` or `marathon` expose `lastUpdated`?

Reply:
The local universe build is run by customers who need a customized, local version of the Universe that only contains the packages they need.   It is also possible to add your own site-local or customized packages using this procedure.   At any rate, since the current state of the master is broken, I'd like to suggest we merge this change and then have a separate discussion about how to best implement the 'lastUpdated' field.

Reply:
+1 if this change resolves COPS-5402 then let's merge and get back to a working state

Reply:
Thanks!